src/error.rs

```diff
@@ -189,6 +190,9 @@ struct ErrorImpl {
 #[doc(hidden)]
 #[derive(Debug)]
 pub enum ErrorCode {
+    /// Control character found while parsing a string.
+    ControlCharacterWhileParsingString,
```

Comment:
Please move this next to the variant you think is most similar to it -- maybe the unicode one.

Reply:
Done.  Moved just after `InvalidUnicodeCodePoint`.

Reply:
Doesn't look like this has been moved yet.